packages/toolpad-app/next.config.mjs

```diff
@@ -92,7 +92,8 @@ const sentryWebpackPluginOptions = {
 
   silent: true, // Suppresses all logs
 
-  dryRun: !process.env.TOOLPAD_SENTRY_DSN,
+  dryRun: true,
+  sourceMapReference: false,
```

Comment:
Can't seem to find docs on this, what does it do?

Reply:
From https://github.com/getsentry/sentry-webpack-plugin#options:
`sourceMapReference` 
```Determines whether sentry-cli should attempt to link minified files with their corresponding maps. By default, it will match files and maps based on name, and add a Sourcemap header to each minified file for which it finds a map. Can be disabled if all minified files contain sourceMappingURL. Defaults to true.```

Maybe it's not worth disabling though, I just thought maybe it'll help Sentry have to do less work, but it's probably not significant...